history-of-quakers/modernized/21-chapter-19.adoc

```diff
@@ -1642,24 +1680,34 @@ this is that saving word of grace which I commend you unto;
 and that God may give you a heart inclinable to believe and obey the truth,
 is the desire of your faithful friend,
 
-R+++.+++ BARCLAY.
+[.signed-section-signature]
+R+++.+++ Barclay.
 
+[.signed-section-context-close]
 From the prison of Aberdeen, in Scotland,
 where I am confined for the sake of the testimony of Jesus.
-November 24,1676.
+
+[.signed-section-context-close]
+November 24, 1676.
 
 This letter, a year ago,
```

Comment:
I'm not sure if this last part should look like this or not.

Reply:
Yeah, this is strange, i'm not sure how i would mark this up either.  what you did looks good in the PDF, so let's leave it how you did it.